server/server/views/profile.py

```diff
@@ -0,0 +1,49 @@
+import json
+import transaction
+
+from pyramid.config import Configurator
+from pyramid.response import Response
+from pyramid.view import view_config
+
+from server.models import get_dbsession
+
+from server.models import Base
+from server.models.user import User
+from server.models.user import users_to_json
+from server.models.user import datetime_conv, AlchemyEncoder
+
+@view_config(route_name='users.view', renderer='json')
+def get_users(request):
+    qry = get_dbsession().query(User).all()
+    return json.loads(json.dumps(qry, cls=AlchemyEncoder))
+
+@view_config(route_name='users.create', renderer='json')
+def create_user(request):
+    json_str = request.json_body
+    json_encoded = json.dumps(json_str)
+    json_decoded = json.loads(json_encoded)
+    user = User(**json_decoded)
+    get_dbsession().add(user)
+    transaction.commit()
+    return {'status': 'Success!'}
+
+@view_config(route_name='users.read', renderer='json')
+def read_user(request):
+    qry = get_dbsession().query(User).filter(User.id==request.matchdict['user_id']).first()
```

Comment:
There is method in SQLAlchemy to get model by id: http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.get

Reply:
Also what if id is not present in DB? You need to return correct HTTP 404 code in this case.